pupil_src/shared_modules/surface_tracker/__init__.py

```diff
@@ -0,0 +1,63 @@
+"""
+(*)~---------------------------------------------------------------------------
+Pupil - eye tracking platform
+Copyright (C) 2012-2018 Pupil Labs
+
+Distributed under the terms of the GNU
+Lesser General Public License (LGPL v3.0).
+See COPYING and COPYING.LESSER for license details.
+---------------------------------------------------------------------------~(*)
+"""
+
+import collections
+from enum import Enum
+
+import numpy as np
+
+Square_Marker_Detection = collections.namedtuple(
+    "Square_Marker_Detection", ["id", "id_confidence", "verts_px", "perimeter"]
+)
+
+
+class Heatmap_Mode(Enum):
+    WITHIN_SURFACE = "Gaze within each surface"
+    ACROSS_SURFACES = "Gaze across different surfaces"
+
+
+class _Surface_Marker_Aggregate(object):
+    """
+    A Surface Marker Aggregate is located in normalized surface space, unlike regular Markers which are
```

Comment:
Thinking about this, this class does actually not require the marker detections to live in normalized surface space, it could be any space. I will push a version that follows your suggestion, but does not specify the coordinates space.